addons/medical_status/XEH_postInit.sqf

```diff
@@ -2,3 +2,24 @@
 
 // Handle pain changes on injury
 [QEGVAR(medical,injured), LINKFUNC(adjustPainLevel)] call CBA_fnc_addEventHandler;
+
+[QGVAR(addInventoryActions), LINKFUNC(addInventoryActions)] call CBA_fnc_addEventHandler;
+
+// Add inventory and open backpack actions to units
+if (hasInterface) then {
+    ["CAManBase", "init", LINKFUNC(addInventoryActions), true, [], true] call CBA_fnc_addClassEventHandler;
+
+    // Respawn is called locally
+    ["CAManBase", "respawn", {
```

Comment:
I initially added this outside of the `hasInterface` scope, as I figured there might be a chance that, for some reason, a player is local to a dedicated server/hc for a while and that the `respawn` EH is called from there, but as the event hasn't been added it doesn't fire.
Is this an unfounded concern of mine? I think the chances of it happening are slight if not remote, but wasn't sure how to handle it.

Reply:
"playable" slots that have AI in them will respawn

Reply:
> edit: so probably move it to top level and just let the hasInterface in addInventoryActions handle it

That was the intention initially.